src/groupeddataframe/splitapplycombine.jl

```diff
@@ -837,22 +910,12 @@ function (agg::Aggregate{typeof(length)})(incol::AbstractVector, gd::GroupedData
     end
 end
 
-function do_f(f, x...)
```

Comment:
Why isn't this needed anymore? I think the trick was to put the return type check in a function that gets specialized on the function and input types for efficiency (to avoid doing checks in type-unstable code).

Reply:
We now do all the checks in `wrap_table` and `wrap_row` (which I had to add anyway). I think it is better to keep the checks in one place. I have not noticed performance impact of these checks. If `f` is not type table it will not be optimized out anyway I think.

But maybe I am missing something here?

Reply:
Probably not, I just wanted to check that we don't miss a performance trick.